src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c

```diff
@@ -998,12 +1019,12 @@ static inline int process_ert_rq(struct xocl_ert_user *ert_user)
 
 		//sched_debug_packet(epkt, epkt->count+sizeof(epkt->header)/sizeof(u32));
 
-		if (cmd_opcode(ecmd) == OP_CONFIG && !ert_user->polling_mode) {
-			for (i = 0; i < ert_user->num_slots; i++) {
-				xocl_intc_ert_request(xdev, i, ert_user_isr, ert_user);
-				xocl_intc_ert_config(xdev, i, true);
-			}
-
+		if (cmd_opcode(ecmd) == OP_CONFIG) {
+			ert_user->is_configured = false;
+			if (ert_user->polling_mode)
+				ert_intc_enable(ert_user, false);
+			else
+				ert_intc_enable(ert_user, true);
```

Comment:
why these are not done in ert_post_process() in CONFIG op code case?

Reply:
it could be, the configuration should be activated once the config cmd is completed and we should change the interrupt and it's rountine setting at the point.

Reply:
config the interrupt and isr in post_process => we use the previous config cmd setting to see if the config cmd is completed or not, even it might change

config the interrupt and isr in process_rq => The setting affects immediately